index.md

```diff
@@ -1,5 +1,50 @@
 # Executable Tutorials Repo setup best practices
 
+## How to Run The Code Yourself
```

Comment:
This was probably a mistake from my rebase. It's been merged so another rebase on `main` will fix.